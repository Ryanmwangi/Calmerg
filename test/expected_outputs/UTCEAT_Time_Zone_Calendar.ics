BEGIN:VCALENDAR
PRODID:-//Your Product ID//EN
VERSION:2.0
NAME:UTCEAT Time Zone Calendar
BEGIN:VEVENT
UID:20231108T100000Z-001@example.com
SUMMARY:UTC_Event UTC Event
DTSTART:20231108T130000
DTEND:20231108T140000
END:VEVENT
BEGIN:VEVENT
UID:20231108T090000+0300-001@example.com
SUMMARY:EAT_Event EAT Event
DTSTART:20231108T090000
DTEND:20231108T100000
END:VEVENT
END:VCALENDAR