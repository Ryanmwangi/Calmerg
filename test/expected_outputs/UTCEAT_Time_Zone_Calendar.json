{
  "linkGroupName": "UTCEAT Time Zone Calendar",
  "calendars": [
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\test\\test_calendars\\utc_time_zone_event.ics",
      "prefix": "UTC_Event",
      "override": false
    },
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\test\\test_calendars\\eat_time_zone_event.ics",
      "prefix": "EAT_Event",
      "override": false
    }
  ]
}